python_ta/.pylintrc

```diff
@@ -5,9 +5,7 @@
 # Default max amount of messages for reporter to display.
 pyta-number-of-messages = 5
 
-# Set whether to use the pycodestyle and type-checking checkers.
-# Anything for a True/False, <yn> value should use in config file: y, yes, n, no
-pyta-pep8 = yes
+# Set whether to use the pycodestyle type-checking checkers.
```

Comment:
Remove the word "pycodestyle" from this sentence, and also the word "checkers" should now be singular.